pkg/kademlia/config.go

```diff
@@ -44,9 +45,7 @@ const (
 type Config struct {
 	BootstrapAddr string `help:"the kademlia node to bootstrap against" default:"bootstrap-dev.storj.io:8080"`
 	DBPath        string `help:"the path for our db services to be created on" default:"$CONFDIR/kademlia"`
-	// TODO(jt): remove this! kademlia should just use the grpc server
-	TODOListenAddr string `help:"the host/port for kademlia to listen on. TODO(jt): this should be removed!" default:"127.0.0.1:7776"`
-	Alpha          int    `help:"alpha is a system wide concurrency parameter." default:"5"`
+	Alpha         int    `help:"alpha is a system wide concurrency parameter." default:"5"`
```

Comment:
I personally don't understand this description. Can we make it more specific? It can be easily confused with upload/download concurrency, or something else.

Reply:
hmm ok Do you have a description that would be more clear?

Reply:
If you can explain what it is about, I can come up with something.

Reply:
"degree of parallelism in network calls"